### ISSUE TITLE
[Feature Request] Multi-objective optimization

### ISSUE THREAD

**Issue by BenikaH** [DimaKudosh/pydfs-lineup-optimizer#373]
Hi @DimaKudosh. Long-time user and appreciate the framework you've built. How hard would it be or is it possible to add multi-objective capabilities to the optimizer? I noticed you added the Mixed-Integer Programming solver, can't wait to try it out. I'm looking to see if I can add multiple objectives here. Thanks again.

**Comment by DimaKudosh** [comment]
What can be the second objective? Ownership? Some solvers support it but not sure how it will work, didn't experiment with multiple objectives

**Comment by BenikaH** [comment]
Yes, I was thinking Ownership as the main secondary objective. As opposed to making it a constraint.
